recipe/meta.yaml

```diff
@@ -34,19 +35,30 @@ requirements:
     - numpy >=1.12  # [win32 and py36]
     - scipy >=0.17
     - matplotlib >=1.3.1
-    - networkx >=1.8
+    - networkx >=1.8,<2.0
     - pillow >=2.1.0
     - dask-core >=0.15
     - toolz >=0.7.4
     - pywavelets >=0.4.0
     - imageio >=2.1.0
 
 test:
+  requires:
+    - pillow <4.2
+    - nose
   imports:
     - skimage
   commands:
     - conda inspect linkages -p $PREFIX scikit-image  # [not win]
     - conda inspect objects -p $PREFIX scikit-image  # [osx]
+    - set MPLCONFIGDIR .  # [win]
+    - set QT_QPA_PLATFORM offscreen  # [win]
+    - set MPLBACKEND Qt4agg  # [win]
+    - MPL_CONFIG_DIR=.  # [not win]
+    - QT_QPA_PLATFORM=offscreen  # [not win]
+    - MPLBACKEND=Qt4agg  # [not win]
+    - echo 'backend.qt : PyQt4' > matplotlibrc
```

Comment:
Would put this line in quotes (everything after the `-`). I think the `:` is getting treated as YAML hence the error message.